setup.py

```diff
@@ -8,8 +8,8 @@
     include_package_data=True,
     install_requires=[
         'cached_property',
-        'django>=1.8,<1.12',
-        'django-mptt',
+        'django==1.11',
+        'django-mptt==0.8.7',
```

Comment:
Sounds like 0.9 caused the issue, so we'll want to exclude that. For maximum compatibility with downstream projects, though, we'll want to expand this version span. Does `django-mptt~=0.8.3` work? That's should match 0.8.3 (the version available when we added this dependency) through 0.8.7.

Reply:
I'll test and make another commit tonight.

Reply:
So on my ubuntu 16.04 machine, running Python 3.6

This is just building the docker image
```docker build . -t eregs/core```

django==1.9 works on mptt>=0.8.3,<=0.8.7
django==1.10 works on mptt>=0.8.6,<=0.8.7
django==1.11 works on mptt>=0.8.6,<=0.8.7

I'm not too familiar with Python setuptools, is there a way to make conditional library versioning?

Reply:
Womp womp. Thanks so much for testing all of that! Looks like we'll get the max coverage with
```
django >= 1.10, <1.12
django-mptt ~= 0.8.6
```
I've opened #87 for removing mptt going forward.